resources/scss/componets/05_forms.scss

```diff
@@ -0,0 +1,11 @@
+.form-group{
+    .form-control{
+        font-size: 12px;
+        width: 100%;
+        padding: 15px;
+        background: $news-bg;
+        box-shadow: none;
+        outline: none;
+        border: 0;
+    }
+}
```

Comment:
Not like this. It will affect all over the site if we use **.form-group** class
set a form parent class like 

.contact-form
.inline-form etc.

Reply:
Fixed